src/modules/data/blocks/ticket/utils.js

```diff
@@ -4,12 +4,17 @@
 import { string, globals } from '@moderntribe/common/utils';
 
 const { settings, priceSettings, tickets: ticketsConfig } = globals;
+/**
+ * Internal dependencies
+ */
+import { getDefaultProvider, getTicketProviders } from '@moderntribe/tickets/data/blocks/ticket/selectors';
```

Comment:
I personally feel that using selectors in utils is a bit of an anti-pattern, but since these selectors here are only grabbing straight from global configs, this is OK. Just something to be aware of in other cases.